### DISCUSSION TITLE
Treeview bindings needs some help

### DISCUSSION THREAD

**Discussion by JankovicsSandor** [AvaloniaUI/Avalonia#6410] [Q&A] [answered]
Hi.

I am trying to figure out how to capture the expanded event of a treeview node item.

I am trying to implement a file comparator based on an input. Basically I have two treeviews next to each other and I would like to synchronise the folder opening and closing of the two sides. In case someone opens a folder it should open on the other side as well. Capturing an expandd event results in a search in the other tree and setting the expanded property of that node. 

I would like to ask also how to bind the expanded-ness of a node to the viewmodel?

My code rignt now
```<TreeView Items="{Binding WindowsList}" Grid.Row="1" Grid.Column="0" AutoScrollToSelectedItem="True" BorderBrush="Gray" BorderThickness="2">
      <TreeView.DataTemplates>
        <TreeDataTemplate DataType="models:PackageItem" ItemsSource="{Binding Children}" >
          <StackPanel Orientation="Horizontal" Spacing="5">
            <Image Source="{Binding Path=Icon,Converter={StaticResource pathToAssetConverter}}" Width="20" Height="20"/>
            <TextBlock Text="{Binding Name}"/>
          </StackPanel>
        </TreeDataTemplate>
      </TreeView.DataTemplates>
    </TreeView>
```

**Comment by jp2masa** [comment, marked as answer]
Something like this should work:

```xaml
<TreeView.Styles>
    <Style Selector="TreeViewItem">
        <Setter Property="IsExpanded" Value="{Binding IsExpanded}" />
    </Style>
</TreeView.Styles>
```

**Reply by JankovicsSandor** [reply]
Thanks it works
